web/index.php

```diff
@@ -30,7 +30,7 @@
 $ffFactory->enablePageCache();
 
 $ffFactory->setLogger( call_user_func( function() use ( $ffFactory ) {
-	$logger = new Logger( 'WMDE Fundraising Frontend logger' );
+	$logger = new Logger( 'index.php logger' );
```

Comment:
The actual logging is not done in index.php but at other places that use the logger, so calling it index.php is a bit too confusing fir my taste.

The name given here is used as the "channel" designation in the log messages. So the `logger` suffix can be dropped IMO. Why not just call it `Fundraising Frontend` ?